api/src/api/emailCheckRouter.ts

```diff
@@ -0,0 +1,34 @@
+import { UserDataClient } from "@domain/types";
+import type { LogWriterType } from "@libs/logWriter";
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+
+export const emailCheckRouterFactory = (userDataClient: UserDataClient, logWriter: LogWriterType): Router => {
+  const router = Router();
+
+  router.post("/emailCheck", async (req, res) => {
```

Comment:
[sand] If you do opt to move this under `userRouterFactory` I'd vote for also nesting the endpoint like this:
```suggestion
  router.post("/users/emailCheck", async (req, res) => {
```